<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1089</width>
    <height>656</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1253</width>
    <height>804</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QGraphicsView" name="my_screen"/>
      </item>
      <item row="0" column="1">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLineEdit" name="set_X"/>
        </item>
        <item>
         <widget class="QLineEdit" name="set_Y"/>
        </item>
        <item>
         <widget class="QPushButton" name="set_ship_btn">
          <property name="text">
           <string>Ustaw statek</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="connect_btn">
          <property name="text">
           <string>Połącz z:</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLineEdit" name="ip_addr_field"/>
          </item>
          <item>
           <widget class="QLineEdit" name="port_field"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QPushButton" name="shot_btn">
            <property name="text">
             <string>Strzał</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="move_btn">
            <property name="text">
             <string>Przesuń</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Y</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="x_field"/>
          </item>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>X</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="y_field"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QPushButton" name="save_btn">
            <property name="text">
             <string>Zapisz</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="read_file_btn">
            <property name="text">
             <string>Wczytaj</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QPushButton" name="back_btn">
            <property name="text">
             <string>Wstecz</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="next_btn">
            <property name="text">
             <string>Dalej</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="gen_rand_btn">
          <property name="text">
           <string>Random</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="ai_btn">
          <property name="text">
           <string>Sztuczna Inteligencja</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="clear_btn">
          <property name="text">
           <string>Wyczyść pole</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="exit_btn">
            <property name="text">
             <string>Zamknij</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1089</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuINfo">
    <property name="title">
     <string>Info</string>
    </property>
    <addaction name="actionInstruction"/>
   </widget>
   <addaction name="menuINfo"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionInstruction">
   <property name="text">
    <string>Instruction</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
